app/models/project.rb

```diff
@@ -1,4 +1,18 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def challenge_theme
+    challenge.theme
+  end
+
+  def number_of_contestants
+    contestants.count
+  end
+
+  def average_years_of_experience
+    average_years = contestants.average(:years_of_experience)
```

Comment:
You don't need to save this query to a variable `average_years`. You could just have the query.